gradle/verification-metadata.xml

```diff
@@ -1417,6 +1418,11 @@
             <pgp value="4f7e32d440ef90a83011a8fc6425559c47cc79c4"/>
          </artifact>
       </component>
+      <component group="javax.servlet" name="servlet-api" version="2.4">
```

Comment:
Where does that come from? We only had servlet API 3.1.0 before. Can we exclude this transitive dependency from whichever dependency which brings this in now?

Reply:
It's a transitive dependency of the new jcifs artifact. I've exlcuded it in 527617ac78e98a72d3df96c55fd7db161fb6b786.